src/entities/purchase-result.ts

```diff
@@ -19,4 +19,15 @@ export interface PurchaseResult {
    * The operation session id of the purchase.
    */
   readonly operationSessionId: string;
+
+  /**
+   * The email of the customer who made the purchase.
+   */
+  readonly customerEmail: string | null;
```

Comment:
I think this makes sense... I was thinking whether there would be any PII concern, but from what I can think of, any concerns would already happen between introducing the email and the payment details. Also, I think this could be useful for devs in some situations yeah.